calculator.js

```diff
@@ -1,11 +1,26 @@
-console.log('Welcome to My Calculator');
+function calculate() {
 
+    let x = document.getElementById("num1").value;
+    let y = document.getElementById("num2").value;
+
+    var checkboxAdd = document.getElementById('add').checked;
+    var checkboxSubtract = document.getElementById('subtract').checked;
```

Comment:
We were more looking for a drop down box for this option.  Cleaner code and a switch statement can be attached as well.  The if else statement works just fine this would just be a better scenario for a switch statement.